apollo-router/src/query_planner/caching_query_planner.rs

```diff
@@ -122,6 +131,23 @@ where
                                 inner_e
                             );
                         }
+                    } else if let QueryPlannerError::SpecError(e) = re.deref() {
+                        let error_key = match e {
+                            SpecError::ParsingError(_) => "## GraphQLParseFailure\n",
```

Comment:
Just checking, have `## GraphQLParseFailure` and `## GraphQLValidationFailure` been confirmed to be valid values for usage reporting?

Reply:
https://github.com/apollographql/apollo-server/blob/270d8c25d325d584b7b446af472e55abd9e72448/packages/apollo-server-core/src/plugin/usageReporting/plugin.ts#L646

Reply:
yup, it should not move (and if I do not specify this exact value here a telemetry test will fail so we indirectly check for it)